core/src/main/java/io/keen/client/java/ScopedKeys.java

```diff
@@ -63,7 +63,11 @@
      */
     public static String encrypt(String apiKey, Map<String, Object> options)
             throws ScopedKeyException {
-        return encrypt(KeenClient.client(), apiKey, options);
+        if (apiKey.length() == 64) {
+            return encrypt(KeenClient.client(), apiKey, options);
+        } else {
+            return oldEncrypt(KeenClient.client(), apiKey, options);
```

Comment:
We need to leave one as `encrypt` for API contract reasons. But I agree that `oldEncrypt` is a bad name so I've fixed that.